manifests/h/Huawei/IdeaShareProject/6.19.0.39/Huawei.IdeaShareProject.installer.yaml

```
@@ -0,0 +1,15 @@
+# Created with YamlCreate.ps1 v2.4.1 $debug=QUSU.CRLF.7-4-1.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: Huawei.IdeaShareProject
+PackageVersion: 6.19.0.39
+InstallerType: nullsoft
+Scope: machine
+UpgradeBehavior: install
+Installers:
+- Architecture: x86
+  InstallerUrl: https://res-static.hc-cdn.cn/cloudbu-site/china/zh-cn/prudout/ec/share/V7.02.1.08Patch/Project_Setup.exe
```

Hi @SpecterShell, 

I found a slightly different `InstallerUrl`: 

```suggestion
  InstallerUrl: https://res-static.hc-cdn.cn/cloudbu-site/china/zh-cn/prudout/ec/share/V7.02.1.08Patch/IdeaShare_Setup.exe
```

Should it be used instead? If not, can you help me to find this `InstallerUrl`?

---

Found it, thank you. 🙂